rasa/core/evaluation/marker.py

```diff
@@ -18,12 +19,12 @@ class AndMarker(OperatorMarker):
     @staticmethod
     def positive_tag() -> Text:
         """Returns the tag to be used in a config file."""
-        return "and"
+        return "AND"
```

Comment:
Does this mean we're capitalising the operators in the config file?

Reply:
yes, "upper-case operators (that was the choice for slot mappings until that got removed - but think it's nice to make it easier to differentiate)"

Reply:
So, do you think this is easier to read/write or not?

Reply:
I got used to the lowercase one but I think they are both fine. No strong preference.

Reply:
@usc-m how about you?

Reply:
I prefer the lowercase one as well since it's less effort to type and reads visually a bit nicer to me, but it's not a strong preference either

Reply:
the typing issue is a good point ... looks like all in all lower case wins this, will revert :)